### PR TITLE
Add active plan status to side bar

### PR DESCRIPTION
**Pull request by MWargelin**
Adds status information about the active plan to the status bar.

The design created for this feature can be found here: https://app.asana.com/0/1206017643443542/1208607227000757/f

### CODE REVIEW

File: admin_site/templates/wagtailadmin/base.html [hunk 1]
```diff
@@ -18,7 +18,23 @@
...
         <img src="{% static 'images/kausal-avatar-dark-green.svg' %}" alt="Kausal Logo" width="80" />
     {% endif %}
-    <div class="active-plan">{{ active_plan.name }}</div>
+
+    <div class="active-plan-info">
+        <div class="active-plan-name">{{ active_plan.name_i18n }}</div>
+        <div class="active-plan-status">
+            {% if active_plan.actions_locked %}
+                <div class="active-plan-locked">{% icon name="lock" %}</div>
+            {% endif %}
```
Comment (MWargelin) [inline admin_site/templates/wagtailadmin/base.html]: Did I get the logic right here, i.e. is `active_plan.actions_locked` what was meant by the plan being locked?

Comment (tituomin) [reply admin_site/templates/wagtailadmin/base.html]: Actually we haven't implemented the locking yet. `actions_locked` was a good guess but we plan to have some feature which locks everything, not just actions.

File: admin_site/templates/wagtailadmin/base.html [hunk 2]
```diff
@@ -18,7 +18,23 @@
...
+            {% if active_plan.version_name %}
+                <div class="active-plan-version-name">{{ active_plan.version_name }}</div>
+            {% endif %}
+            {% if active_plan.is_live %}
+                <div>
+                    <span class="active-plan-live">live</span>
+                </div>
+            {% endif %}
```
Comment (tituomin) [inline admin_site/templates/wagtailadmin/base.html]: This is good, but I realized there is a bug in Plan.is_live. It should compare the date to the current date. But not a problem wrt this PR.

Another thing is that I think if it's not live, we want to display that info as well, we could use the term "draft" for the opposite state. Or "internal" is another option.

File: admin_site/templates/wagtailadmin/base.html [hunk 3]
```diff
@@ -18,7 +18,23 @@
...
+        <div class="active-plan-status">
+            {% if active_plan.actions_locked %}
+                <div class="active-plan-locked">{% icon name="lock" %}</div>
+            {% endif %}
+            {% if active_plan.version_name %}
+                <div class="active-plan-version-name">{{ active_plan.version_name }}</div>
+            {% endif %}
+            {% if active_plan.is_live %}
+                <div>
+                    <span class="active-plan-live">live</span>
```
Comment (tituomin) [inline admin_site/templates/wagtailadmin/base.html]: We should localize these UI strings.

File: admin_site/templates/wagtailadmin/base.html [hunk 4]
```diff
@@ -18,7 +18,23 @@
     {% else %}
         <img src="{% static 'images/kausal-avatar-dark-green.svg' %}" alt="Kausal Logo" width="80" />
     {% endif %}
-    <div class="active-plan">{{ active_plan.name }}</div>
+
+    <div class="active-plan-info">
+        <div class="active-plan-name">{{ active_plan.name_i18n }}</div>
+        <div class="active-plan-status">
+            {% if active_plan.actions_locked %}
+                <div class="active-plan-locked">{% icon name="lock" %}</div>
```
Comment (tituomin) [inline admin_site/templates/wagtailadmin/base.html]: We probably want an aria-attribute for accessibility, some verbal description of the state in addition to the icon

File: admin_site/templates/wagtailadmin/base.html (hunk 1 above)
Comment (tituomin) [reply admin_site/templates/wagtailadmin/base.html]: It's good to have this for testing though, and we might just comment it out for now, before deploying.

**Review by tituomin — CHANGES_REQUESTED** [review]
I will still test how this looks. Left some requests.

File: admin_site/templates/wagtailadmin/base.html (hunk 1 above)
Comment (MWargelin) [reply admin_site/templates/wagtailadmin/base.html]: I thought it was something like that, good that that was cleared. I commented the code out for now with a TODO message.

File: admin_site/templates/wagtailadmin/base.html (hunk 2 above)
Comment (MWargelin) [reply admin_site/templates/wagtailadmin/base.html]: Good catch with the `is_live` bug. Seemed like a small thing so fixed it here anyway, is this what you meant? 43ee9803182fb97637651f2b8156a16b111c866f

I changed this now to print "draft" if the plan is not live.

File: admin_site/templates/wagtailadmin/base.html (hunk 3 above)
Comment (MWargelin) [reply admin_site/templates/wagtailadmin/base.html]: localised the strings :+1:

File: admin_site/templates/wagtailadmin/base.html (hunk 4 above)
Comment (MWargelin) [reply admin_site/templates/wagtailadmin/base.html]: I'm not very familiar with aria and accessibility in general, I added `aria-label` that says `locked plan` (translated), does that sound right?

**Comment by MWargelin** [comment]
@tituomin I believe I've now implemented all the requested changes, is this good to go now?

**Pull request merged**